csp_billing_adapter_k8s/plugin.py

```
@@ -92,7 +108,7 @@ def save_cache(config: Config, cache: dict):
             return None  # Already exists
         else:
             log.error(f'Failed to save cache: {str(error)}')
-            raise
+            _re_raise_api_exception(error)
```

So technically this is raising a new exception, with a potentially new error message, whereas previously this was re-raising the original exception, which would come with the original traceback info...
Raising a new exception here means that we lose that original traceback data, and start with a traceback from the point where we raise the new exception...
Are we happy to lose the additional traceback data, from a debugging/support perspective?
Or would it be better to raise the new exception "from" the old exception, which, IIRC, would retain the original exception's traceback?

---

Not sure if it is of any help but raised from the error anyway.